features/support/helpers.js

```diff
@@ -11,7 +13,8 @@ var helpers = {
       .replace(/^\s+/g, '')
       .replace(/\s+$/g, '')
       .replace(/[ \t]+\n/g, '\n')
-      .replace(/\d+m\d{2}\.\d{3}s/, '<duration-stat>');
+      .replace(/\d+m\d{2}\.\d{3}s/, '<duration-stat>')
+      .replace(/\//g, path.join('/'));
```

Comment:
Can `path.join('/')` just be path.sep? [docs](https://nodejs.org/api/path.html#path_path_sep)